app/lib/features/tasks/widgets/due_chip.dart

```diff
@@ -81,7 +81,7 @@ class _DueChipState extends State<DueChip> {
       label: Text(
         // FIXME: tooltip to show the full date?
         label ??
-            '${L10n.of(context).due('')}: ${DateFormat(DateFormat.YEAR_MONTH_WEEKDAY_DAY).format(dueDate!)}',
+            '${L10n.of(context).due}: ${DateFormat(DateFormat.YEAR_MONTH_WEEKDAY_DAY).format(dueDate!)}',
```

Comment:
this should be an input as well.
```suggestion
           L10n.of(context).dueDate(DateFormat(DateFormat.YEAR_MONTH_WEEKDAY_DAY).format(dueDate!)),
```

And arguably the dueDate format might also be something we want to localize ... but I suppose that happens in the DateFormat-lib rather than us giving the format in the language-file...?!?

Reply:
Created separate ticket to manage this.
https://github.com/acterglobal/a3/issues/1577